docs/modeling-your-data/modeling-your-data-with-dbt/package-features/table-grants/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Table Grants"
+description: "Details for granted access to our tables to other users/roles"
+sidebar_position: 30
+---
+
+:::tip
+
+This functionality requires dispatching to our macros over dbt core, see how to do this on the [dispatch setup](/docs/modeling-your-data/modeling-your-data-with-dbt/package-mechanics/dispatch/index.md) page.
+
+:::
+
+## Availability
+:::tip
+
+Any package can make use of the table grants feature provided by the `snowplow__grant_select_to` variable if you are using at least version 0.16.2 of snowplow-utils, even if the variable is not listed in the configuration. Granting usage on schemas however requires specific package versions.
+
+:::
+
+| Package | Minimum Required Version |
+|---------|--------------------------|
+| Unified | 0.3.0 |
+
+Note that this feature is not supported for BigQuery due to the different approach to permissions they take via IAM roles.
+
+## Usage
+### Granting select on tables
+To grant `select` on all tables created within our package, you can provide a list of the users/roles in the `snowplow__grant_select_to` variable e.g.
+
+```yaml title=dbt_project.yml
+vars:
+  snowplow_<package_name>:
+    snowplow__grant_select_to: ['myuser1', 'myuser2']
+```
+Note that these user/role names are case sensitive. Databricks Principals are also supported. If the user does not exist an error will occur when the grant tries to run. This feature is compatible with the built-in [dbt grants](https://docs.getdbt.com/reference/resource-configs/grants) functionality and we will grant to a combination of the two.
+
+:::caution
+
+It is important to scope this to the relevant package, if you set this variable at the top level of your project then all models will have this grant applied.
+
+:::
+
+:::caution
+
+Note this will overwrite any existing grants applied to the table manually in the warehouse.
```

Comment:
Is it a will or a may?

Reply:
Will, it checks against the warehouse and revokes any not in the list or provided the normal dbt way https://github.com/dbt-labs/dbt-adapters/blob/6e765f58d1a15f7fcc15e504916543bd55bd62b7/dbt/include/global_project/macros/adapters/apply_grants.sql#L137